spotlight/evaluation.py

```diff
@@ -96,6 +96,60 @@ def sequence_mrr_score(model, test):
     return np.array(mrrs)
 
 
+def precision_recall_at_k(model, test, k):
+    """
+    Compute Precision@k and Recall@k scores. One score
+    is given for every user with interactions in the test
+    set, representing the Precision@k and Recall@k of all their
+    test items.
+
+    Parameters
+    ----------
+
+    model: fitted instance of a recommender model
```

Comment:
We should make this accept train interactions as well in the same way other metrics do.

Reply:
This is so that we can disregard the model giving high scores to known interactions. This reflects real-world system implementations.